src/type-coercions.md

```diff
@@ -164,8 +164,7 @@ the compiler will provide an implementation of `Unsize<U>` for `T`:
 
 * `[T; n]` to `[T]`.
 
-* `T` to `U`, when `U` is a trait object type and either `T` implements `U` or
-  `T` is a trait object for a subtrait of `U`.
+* `T` to `dyn U`, when `T` implements `U + Sized`
```

Comment:
I don't think `+ Sized` is accurate here; without it, coercions of references won't always work because going from `&T` to `&U` would require that `T` be sized. Unless that's actually how it works?